docs/integrations/cloud-security-monitoring-analytics/opentelemetry/linux-opentelemetry.md

```diff
@@ -0,0 +1,148 @@
+---
+id: linux-opentelemetry
+title: Linux - Cloud Security Monitoring and Analytics - OpenTelemetry
+dashboard: The Sumo Logic app for Linux Cloud Security Monitoring and Analytics - OpenTelemetry provides better understanding of your production environments, and surface relevant insights by tuning out-of-the-box content to align with your security team’s focus
+
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/cloud-security-monitoring-analytics/SecMon_Linux.png')} alt="Thumbnail icon" width="100"/>
+
+The Linux - Cloud Security Monitoring and Analytics - OpenTelemetry is a unified log app that ingests distribution of Linux data to Sumo Logic via OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver). The app's preconfigured dashboards provide insight into user, service, systems, login, and privilege activity, providing a better understanding of your production environments and surface relevant insights by tuning out-of-the-box content to align with your security team's focus.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/PCI-Compliance-For-Linux/OpenTelemetry/Linux-Schematics.png' alt="Linux-Schematics" style={{border: '1px solid black'}} />
+
+## Fields created in Sumo Logic for Linux - Security Analytics
+
+Following tag will be created as part of Linux app installation, if not already present. 
+
+- `sumo.datasource`. Has a fixed value of **linux**.
+
+## Collection configuration and app installation
+
+:::note
+You can skip this section if you have already set up the logs collection through [Linux](/docs/integrations/hosts-operating-systems/opentelemetry/linux-opentelemetry/) or Linux PCI app installation. Additional collection is not required as the logs used by this app are already ingested into Sumo Logic.
+:::
+
+### Step 1: Set up Collector
+
+{@import ../../../reuse/apps/opentelemetry/set-up-collector.md}
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/PCI-Compliance-For-Linux/OpenTelemetry/Linux-Collector.png' alt="Linux-Collector" style={{border: '1px solid black'}} />
+
+### Step 2: Configure integration
+
+In this step, you will configure the yaml required for Linux Collection. The app requires path for system log file based on the Linux version used.
+
+#### Required Logs for Ubuntu
+
+The following logs, located in the `/var/log` folder, are required for using the Sumo Logic app for PCI compliance for Linux with Ubuntu.
```

Comment:
This refers to a different app altogether, PCI Compliance for Linux. Is this intentional?

Reply:
Yes Kim, it is intentional. You can refer this [doc](https://help.sumologic.com/docs/integrations/pci-compliance/opentelemetry/linux-opentelemetry/#step-1-set-up-collector).

Reply:
Shouldn't this be reflected in this doc's title? It says it's called "Linux - Cloud Security Monitoring and Analytics - OpenTelemetry".